src/App.js

```diff
@@ -41,10 +42,20 @@ function App() {
             <button onClick={generateTokenHandler}>Get your list token!</button>
             <p>{token}</p>
             <p>Or enter an existing token to see your list below.</p>
-            <ItemList token={token} onEnterToken={enterTokenHandler} />
+            <ItemList
+              token={token}
+              onEnterToken={enterTokenHandler}
+              dbItems={dbItems}
+              setdbItems={setdbItems}
+            />
           </Route>
           <Route exact path="/add">
-            <Form token={token} onEnterToken={enterTokenHandler} />
+            <Form
+              token={token}
+              onEnterToken={enterTokenHandler}
+              dbItems={dbItems}
+              setdbItems={setdbItems}
```

Comment:
I really appreciate that you kept the names of the variables consistent with the prop names, like `setdbItems={setdbItems}`. Sometimes I see people call the prop one thing, and pass in the intended variable as a different name. The way you wrote this makes it really easy to understand!